crt_portal/cts_forms/tests/integration/report_submission.py

```diff
@@ -0,0 +1,142 @@
+
+import pytest
+
+
+@pytest.mark.only_browser("chromium")
+def test_report_complete_and_valid_submission(page):
+    page.goto("/report")
+    assert page.title() == "Step 1: Contact - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Click input[name="0-contact_first_name"]
+    page.click("input[name=\"0-contact_first_name\"]")
+
+    # Fill input[name="0-contact_first_name"]
+    page.fill("input[name=\"0-contact_first_name\"]", "Testing")
```

Comment:
Also this strikes me as ripe for refactoring into a small helper function.

Reply:
Something like this, as a closure (completely untested):

```python
def input(name, value): return page.fill(f'input[name="{name}]', value)
```

But now I'm thinking about this, maybe this file was autogenerated?

Reply:
Ahhh, YES! A great piece that 💯 should have been included in the PR write-up. All of the selectors and input for this was generated w/ 
```shell
python -m playwright codegen
```

There were only 1 or 2 changes needed from that output (checkbox input) to get this to implementation which I thought was a big win. There are for sure better ways, both in terms of robustness and extensibility, to implement this by hand but for this MVP I wanted to focus on the foundational pieces and not fine tuning this specific test.

Reply:
Trimmed out most of the non-essential remnants from the codegen origins!